test-case/verify-sof-firmware-load.sh

```diff
@@ -20,13 +20,11 @@ source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
 func_opt_parse_option "$@"
 
-if ! alias | grep -q 'Sub-Test'; then
-    # hijack DMESG_LOG_START_LINE which refer dump kernel log in exit function
-    DMESG_LOG_START_LINE=$(sof-get-kernel-line.sh|tail -n 1 |awk '{print $1;}')
-    cmd="sof-kernel-dump.sh"
-else
-    cmd="journalctl --dmesg --no-pager"
-fi
+# hijack DMESG_LOG_START_LINE to dump kernel from file start is not Sub-Test
+# TODO: clean up Sub-Test feature
+alias | grep -q 'Sub-Test' || DMESG_LOG_START_LINE=$(sof-get-kernel-line.sh | tail -n 1 | awk '{print $1;}' )
+
+cmd="journalctl_cmd"
```

Comment:
I'm not 100% sure this has to be in the same PR than the rest.

Reply:
I named this PR as `use journalctl as default kernel log reader`, so I think put all change there is OK.
I split them into different commit. If you want single commit in one PR I am OK with that.